src/submission/validation-clinical/therapy.ts

```diff
@@ -169,13 +169,16 @@ const validateRadiationRecords = async (
     }
 
     // Therapy + Treatment are associated with Radiation
-    const currentTreatmentIsRadiation = treatment_type === 'radiation';
+    const currentTreatmentIsRadiation =
+      Array.isArray(treatment_type) && treatment_type.includes('Radiation therapy');
 
     const submittedTreatmentIsRadiation =
-      submissionTreatmentIdMatch && submissionTreatmentIdMatch.treatment_type === 'radiation';
+      Array.isArray(submissionTreatmentIdMatch?.treatment_type) &&
+      submissionTreatmentIdMatch?.treatment_type.includes('Radiation therapy');
 
     const storedTreatmentIsRadiation =
-      storedTreatmentIdMatch?.clinicalInfo?.treatment_type === 'radiation';
+      Array.isArray(storedTreatmentIdMatch?.clinicalInfo?.treatment_type) &&
+      storedTreatmentIdMatch?.clinicalInfo?.treatment_type.includes('Radiation therapy');
```

Comment:
suggestion that we change this for a single object lookup
something like: 
```javascript
const storedTreatmentType = storedTreatmentIdMatch?.clinicalInfo?.treatment_type;
const storedTreatmentIsRadiation =
   Array.isArray(storedTreatmentType) &&
   storedTreatmentType.includes('Radiation therapy');
   ```

Reply:
Sure, will do the same for `submittedTreatmentType`